src/qml/LayerTreeItemProperties.qml

```diff
@@ -45,6 +45,9 @@ LayerTreeItemProperties {
   }
 
   function isTrackingButtonVisible() {
+    if ( ! index )
+      return false
+
```

Comment:
I think we could drop this whole file.
@m-kuhn @nirvn  ?

Reply:
I don't know, why?

Reply:
The layer tree is not used anymore, right?

Reply:
This is the layer tree properties, that is shown when a layer tree node is long/double pressed, I don't think we should drop it? Nevertheless, I am not sure why there are two files with that name in qml and qml/ui thought, but both are actively used.

Reply:
If we can get rid of the .ui file, that'd be great. I can't explain why, but it seems to trigger the last remaining bit of https://github.com/opengisch/QField/issues/987 .

Reply:
Can we get rid of this file in a seperate PR, just to keep this one small and focused?